types/are.na/index.d.ts

```diff
@@ -0,0 +1,621 @@
+// Definitions by: Hunter Caron <https://github.com/huntercaron>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 4.1
+
+/**
+ * Timestamp in ISO 8601 format, eg. "2020-06-20T15:59:04.796Z"
+ */
+type Timestamp = string;
+
+/**
+ * Parameter type for API functions
+ */
+interface Params {
+    [key: string]: any;
+}
+
+/**
+ * Parameter type for API functions with pagination support
+ */
+interface PaginationParams extends Params {
+    page: number;
+    per: number;
+}
+
+/**
+ * Arena class, organized hierarchically as nested objects
+ */
+declare class Arena {
+    /**
+     * Arena class, organized hierarchically as nested objects
+     *
+     * ```js
+     * // Request for a channel with slug of "arena-influences"
+     * const arena = new Arena();
+     * arena.channel("arena-influences").get()
+     * ```
+     * @param accessToken - Your are.na API access token
+     * @param baseURL - Base URL to make requests on (default: https://api.are.na/v2/)
+     */
+    constructor(config?: { accessToken?: string; baseURL?: string });
+
+    /**
+     * Base object for channel related API requests
+     *
+     * ```js
+     * // Basic usage
+     * arena.channel("arena-influences").get()
+     * ```
+     *
+     * @param indentifier - Channel slug or id, such as "arena-influences" or 275
+     * @param params - Additional parameters
+     */
+    channel(
+        indentifier: string | number,
+        params?: Params,
+    ): {
+        /**
+         * Get the channel as an Object. Gets a list of public channels if slug/id not specified. Supports pagination.
+         */
+        get(params?: PaginationParams): Promise<Arena.Channel>;
+        /**
+         * Limited view of the channel
+         */
+        thumb(params?: Params): Promise<Arena.Channel>;
+        /**
+         * Get all of the connections of the channel (channels where this channel is connected). Supports pagination.
+         */
+        connections(params?: PaginationParams): Promise<any>;
+        /**
+         * Get all of the channels connected to blocks in the channel. Supports pagination.
+         */
+        channels(params?: PaginationParams): Promise<Arena.Channel[]>;
+        /**
+         * Get the channel's contents only, as an array. Supports pagination.
+         */
+        contents(params?: PaginationParams): Promise<Arena.Block[]>;
+        /**
+         * Get the channel's collaborators. Supports pagination.
+         */
+        collaborators(params?: PaginationParams): Promise<Arena.User[]>;
+        /**
+         * Creates a new channel. Called as or channel().create(title, status). Title is required, status is optional.
+         */
+        create(title: string, status: Arena.ChannelStatus): Promise<void>;
+        /**
+         * Creates a new channel. Called as channel(title).create(status). Title is required, status is optional.
+         */
+        create(status: Arena.ChannelStatus): Promise<void>;
```

Comment:
it seems, in this function is a missing `title` by your documentation but then it is the same as https://github.com/DefinitelyTyped/DefinitelyTyped/pull/51403/files#diff-91c4eda792aace8177f766a6c99725c79a920925a768795e12bd0f3bfdd80ca0R86 so I think that it is unnecessary

Reply:
This function is overloaded with either `(status)` or `(title, status)` as documented inline here https://github.com/ivangreene/arena-js/blob/master/index.js#L103-L104